src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2361,12 +2361,22 @@ public static string LastWriteTimeString(PSObject instance)
                     // non-existing targets on either Windows or Linux.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
New-Item: symlink with absolute/relative path test.Symlink with relative path to existing directory behaves like a directory
```powershell
Expected the value to have type [System.IO.DirectoryInfo] or any of its subtypes, but got /private/tmp/6990029e-f7fb-4110-a26a-b634f5ba29a5/someDirLinkRelative with type [System.IO.FileInfo].
at <ScriptBlock>, /Users/runner/runners/2.169.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1: line 302
302:         Get-Item someDirLinkRelative | Should -BeOfType System.IO.DirectoryInfo
```